.github/workflows/ci.yml

```diff
@@ -2,8 +2,13 @@ name: CI (GitHub)
 on:
   - push
   - pull_request
+permissions: {}
 jobs:
   CI:
+    permissions:
```

Comment:
Is there a benefit in listing permissions under a job instead of a workflow? Should these two files use a consistent approach?

Reply:
The only benefit is that if later somebody adds a new job it will inherit the global permission automatically, so it is safer to have read-only at the top. I made it consistent now.